content/using/running/coturn-guide.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "coturn Operator Guide"
+description = "Instructions for running a coturn server on your star."
+weight = 4
+template = "doc.html"
++++
+
+# coturn Operator Guide
+
+To make voice and video calls through urbit we use a protocol called WebRTC which allows ships to connect directly to each other. Most times one or both ships are behind NATs and firewalls and often they are unable to connect. To get around this, instead of connecting directly to each other the ships can all connect to a TURN server (Transversal Using Relays around NATs) which will relay all the data between all the participants. Since this is relaying a bunch of data there are no free TURN servers on the internet. Thus a nice service that a star can provide is to run a TURN server and allow it's sponsored planets to use it. This guide will show you how to set one up.
+
+The server you want to run is called [coturn](https://github.com/coturn/coturn).
+
+coturn requires a server to run on, a domain, and a certificate. It can run on the same machine as your star or a separate machine. 
+
+## Getting a server, domain, and certificate
+
+- If you need a server or domain follow the [Cloud Hosting guide](https://urbit.org/using/running/hosting) until the section 'Installing Urbit.' Stop there since you don't need to install urbit on the coturn machine.
+- Follow [this guide](https://www.digitalocean.com/community/tutorials/how-to-acquire-a-let-s-encrypt-certificate-using-dns-validation-with-certbot-dns-digitalocean-on-ubuntu-20-04) to install certbot and use it to aquire a certificate. 
```

Comment:
```suggestion
If you need a server or domain follow the [Cloud Hosting guide](https://urbit.org/using/running/hosting) until the section 'Installing Urbit.' Stop there since you don't need to install urbit on the coturn machine. Follow [this guide](https://www.digitalocean.com/community/tutorials/how-to-acquire-a-let-s-encrypt-certificate-using-dns-validation-with-certbot-dns-digitalocean-on-ubuntu-20-04) to install certbot and use it to aquire a certificate.

Note the instructions below assume an Ubuntu server. If your server runs a different distribution, you may need to substitute `apt-get` and `ufw` with your distro's package manager and firewall.
```